### DISCUSSION TITLE
High CPU usage, random frame drops and increased latency

### DISCUSSION THREAD

**Discussion by dragonGod97677** [ppy/osu#31231] [Q&A]
### Type

Performance

### Bug description

I have fresh install of lazer in my PC, when it started importing from stable, the game and my PC lags, and CPU usage is max 100 percent, after the import finished, it doesn't lag anymore and while idle CPU usage is max 100 and there are random frame drops and increased latency for half a second. 
While im playing, it lags, and while on failed screen, the frame drops and high latency is much more noticeable.

My PC Specs:
AMD A6-7480
Inno3D GT 730
2x8GB RAM
250GB Crucial MX500

### Screenshots or videos

[after the import](https://youtu.be/hShxh2v8U00?si=9rMhpHHGm3wR1aO1)
[While importing](https://youtu.be/wajn9bPm52Y?si=i2Jz7zJ8qVnNVREO)
[Idle on failed screen](https://youtu.be/WpEs0hVqLO4?si=rwvf0z9iUS-Td_NJ)

### Version

2024.1219.2-lazer

### Logs

(i have lost the log that while it was importing but here)

https://drive.google.com/file/d/1VonDKuxyAa_IQpukSFI1EjVl7A1fczHi/view?usp=drive_link

**Comment by DarkGradient** [comment]
Hm.. Looks like it's problem with your system.
"after import" and "idle on failed screen"
While importing beatmaps, lagspikes, low fps are expected.
I can suggest - disk cleanup, virus scan, tweak windows ( disable most of telemetry settings ), don't use other apps while playing.

**Reply by dragonGod97677** [reply]
My windows installation is fresh and telemetry service that uses CPU too much is already removed too and i tried with only lazer just open, and same results
